config/core/test-configs.yaml

```diff
@@ -155,6 +155,17 @@ test_plans:
       - passlist: &qemu_labs_filter
           lab: ['lab-baylibre', 'lab-collabora-staging']
 
+  chromeos-flash:
+    pattern: 'chromeos/chromeos-flash-template.jinja2'
+    params:
+      job_timeout: '15'
+      docker_image: 'kernelci/chromeos-flash'
+      chromeos_image: 'chromiumos_test_image.bin'
+      flashing_kernel: https://people.collabora.com/~mgalka/stable/linux-5.10.y/v5.10.95/x86_64/x86_64_defconfig+x86-chromebook/gcc-10/kernel/bzImage
+      flashing_modules: https://people.collabora.com/~mgalka/stable/linux-5.10.y/v5.10.95/x86_64/x86_64_defconfig+x86-chromebook/gcc-10/modules.tar.xz
+      flashing_rootfs: https://storage.kernelci.org/images/rootfs/debian/bullseye-chromeos-flash/20220311.0/amd64/full.rootfs.tar.xz
+      flashing_ramdisk: http://storage.kernelci.org/images/rootfs/debian/buster/20210503.0/amd64/initrd.cpio.gz
```

Comment:
I think these images should eventually be stored on storage.chromeos.kernelci.org

It's probably not a blocker for this PR, but some tidying-up we can do once this is working.